library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -164,6 +230,47 @@ internal class TransactionPayloadFragment :
         }
     }
 
+    private class FileSaverTask(val fragment: TransactionPayloadFragment) :
+        AsyncTask<Triple<Int, Uri, HttpTransaction>, Unit, Boolean>() {
+
+        @Suppress("NestedBlockDepth")
+        override fun doInBackground(vararg params: Triple<Int, Uri, HttpTransaction>): Boolean {
+            val (type, uri, transaction) = params[0]
+            try {
+                val context = fragment.context ?: return false
+                context.contentResolver.openFileDescriptor(uri, "w")?.use {
+                    FileOutputStream(it.fileDescriptor).use { fos ->
+                        if (type == TYPE_REQUEST) {
+                            transaction.requestBody?.byteInputStream()?.copyTo(fos)
+                        } else if (transaction.responseBody != null) {
+                            transaction.responseBody?.byteInputStream()?.copyTo(fos)
+                        } else {
+                            fos.write(transaction.responseImageData)
+                        }
+                    }
+                }
+            } catch (e: FileNotFoundException) {
+                e.printStackTrace()
+                return false
+            } catch (e: IOException) {
+                e.printStackTrace()
+                return false
+            }
+            return true
+        }
+
+        override fun onPostExecute(result: Boolean) {
```

Comment:
I suggest to rename the variable to match its type.

Reply:
I'm not sure I got it 🤔

Reply:
In this particular case we have Task for saving the result into file, so we could name the parameter as `isSaved` or `wasSaved` or `isSuccessful`. After such change the condition below would look like:
```
if (isSaved) {
....
}
```
which looks quite more readable.

Reply:
Got it :) Yup I'm changing it